packages/amplify-graphql-api-construct-tests/src/__tests__/sql-models.test.ts

```diff
@@ -83,10 +85,22 @@ describe('CDK GraphQL Transformer', () => {
     deleteProjectDir(projRoot);
   });
 
-  it('creates a GraphQL API from SQL-based models', async () => {
+  test('creates a GraphQL API from SQL-based models with Secrets Manager Credential Store default encryption key', async () => {
```

Comment:
I note that these tests won't exercise the "secrets created by RDS" flow, only the "secrets manually created" flow, correct? Given that's an officially supported use case I'd be happier if we had a test to exercise that flow. I think it would require a different `backend` though, since the one for the sql-models tests explicitly passes username/password to the RDS instance construct. If you agree, please add in a separate PR rather than increasing the line count on this one :)